src/axes.jl

```diff
@@ -241,7 +241,7 @@ function get_ticks(axis::Axis)
     elseif typeof(ticks) <: Union{AVec, Int}
         # override ticks, but get the labels
         optimal_ticks_and_labels(axis, ticks)
-    elseif typeof(ticks) <: NTuple{2}
+    elseif typeof(ticks) <: NTuple{2, Any}
```

Comment:
It's probably better to not do this, and the rebase this PR onto master before merging (or now, if you want the functionality at this point). I'm fairly sure this will cause git conflicts.